source/simulator/stokes_matrix_free.cc

```diff
@@ -387,80 +387,98 @@ namespace aspect
   template <int dim, int degree_v, typename number>
   void
   MatrixFreeStokesOperators::StokesOperator<dim,degree_v,number>
-  ::local_apply (const dealii::MatrixFree<dim, number>                 &data,
+  ::local_apply (const dealii::MatrixFree<dim, number>                         &data,
                  dealii::LinearAlgebra::distributed::BlockVector<number>       &dst,
                  const dealii::LinearAlgebra::distributed::BlockVector<number> &src,
-                 const std::pair<unsigned int, unsigned int>           &cell_range) const
+                 const std::pair<unsigned int, unsigned int>                   &cell_range) const
   {
-    FEEvaluation<dim,degree_v,degree_v+1,dim,number> velocity (data, 0);
-    FEEvaluation<dim,degree_v-1,  degree_v+1,1,  number> pressure (data, /*dofh*/1);
+    FEEvaluation<dim,degree_v,degree_v+1,dim,number> u_eval(data, 0);
```

Comment:
What's the reason for these name changes?

Reply:
The original names are a little confusing to me, because function FEEvaluation::get_value() returns the value of the corresponding field, not the value of the shape function. So if one calls pressure.get_value(), then the returned value is the pressure itself, which looks weird.